cluster.go

```diff
@@ -267,6 +285,8 @@ func NewCluster(hosts ...string) *ClusterConfig {
 		ConnectTimeout:         600 * time.Millisecond,
 		Port:                   9042,
 		NumConns:               2,
+		HeavyLoadedConnectionThreshold: 512,
+		HeavyLoadedSwitchConnectionPercentage: 20,
```

Comment:
Please run gofmt on this file. It will be annoying for others contributors if gofmt keeps formatting code that they didn't modify.